go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/btcsuite/btcd/chaincfg/chainhash v1.0.1
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0
 	github.com/ethereum-optimism/go-ethereum-hdwallet v0.1.3
-	github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20230920213331-413695cf7906
+	github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20230921150749-b0edabdeac8c
```

Comment:
Would be nice to start versionning the superchain-regsitry. It's a pretty simple codebase, so should be easy to do so.

Reply:
I agree but don't think its a blocker for this merge